README.md

```diff
@@ -23,9 +23,11 @@ Resources
 
 Contribute
 ----------
-- Pull requests are welcome. Please discuss bigger changes in the conference room first.
+- Pull requests are welcome. A good start are [good first issues](https://github.com/dino/dino/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22). 
+Please discuss bigger changes in the [conference room](xmpp:chat@dino.im?join) first.
```

Comment:
IIRC, GitHub does not support linking `xmpp:`-URIs, so this link wouldn't be displayed.